solr/modules/hdfs/src/test/org/apache/hadoop/util/DiskChecker.java

```diff
@@ -25,7 +25,7 @@
 import java.util.UUID;
 import java.util.concurrent.atomic.AtomicReference;
 
-import com.google.common.annotations.VisibleForTesting;
+import org.apache.hadoop.thirdparty.com.google.common.annotations.VisibleForTesting;
```

Comment:
Did Hadoop switch to shading google-annotations? I thought we already had our own copy of it anyway.

Reply:
Guava is shaded by Hadoop. I don't know that it matters where we get the Guava annotations from. We still pull in Guava for the HDFS block cache stuff, but I figured it made more sense to pull in the Hadoop shaded Gauva to keep the `org.apache.hadoop` files closer to the original.